grype/db/v6/distribution/latest.go

```diff
@@ -67,6 +73,43 @@ func NewLatestFromReader(reader io.Reader) (*LatestDocument, error) {
 	return &l, nil
 }
 
+func NewArchive(path string) (*Archive, error) {
```

Comment:
Is this constructor still necessary if the suggestion at https://github.com/anchore/grype-db/pull/437/files#r1857293906 is implemented?